### ISSUE TITLE
Move assets into their own sub-directory

### ISSUE THREAD

**Issue by mplorentz** [prbc/farese#19]
Right now the repo is littered with .jpg and .gif files. I think it would make things easier to work with if we moved those into an "assets" folder. This would of course involve updating all links to them in the HTML. Thoughts?

**Comment by dbirks** [comment]
Yeah, it's pretty ugly. Maybe we can ls all .gif and .jpg files, then use find and sed to loop through them. Like [this](http://askubuntu.com/questions/84007/find-and-replace-text-within-multiple-files) maybe. I can work on that. (Probably before Saturday right?)

**Comment by mplorentz** [comment]
@dbirks oh no I wasn't thinking for Saturday. Just something I thought we should get to eventually. Not high priority at all.

**Comment by dbirks** [comment]
Hi @kingnebby, thanks a lot for offering to do this. If it helps at all, here's what I have so far:

```
#/bin/sh

mkdir assets

for i in {*.gif,*.jpg}; do
    find . -name \*.htm* -exec sed -i "s/$i/.\/assets\/$i/g" {} \;
    mv $i ./assets/.
done
```

The problem with this is that it modifies files in directories as well, so those links end up being broken.

Or you can go about it a completely different way.

**Comment by dbirks** [comment]
Nice, closed with #28.
